src/main/java/com/baidu/hugegraph/structure/auth/Project.java

```diff
@@ -0,0 +1,156 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.baidu.hugegraph.structure.auth;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+import com.baidu.hugegraph.structure.constant.HugeType;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Project extends AuthElement {
+
+    @JsonProperty("project_description")
+    private String description;
+    @JsonProperty("project_target")
+    private String target;
+    @JsonProperty("project_admin_group")
+    private String adminGroup;
+    @JsonProperty("project_op_group")
+    private String opGroup;
+    @JsonProperty("project_name")
+    private String name;
+    @JsonProperty("project_graphs")
+    private List<String> graphs;
+    @JsonProperty("project_graph")
+    private String graph;
+
+    @JsonProperty("project_create")
+    @JsonFormat(pattern = DATE_FORMAT)
+    private Date create;
+    @JsonProperty("project_creator")
+    private String creator;
+    @JsonProperty("project_update")
+    @JsonFormat(pattern = DATE_FORMAT)
+    private Date update;
+
+    public String description() {
+        return description;
+    }
+
+    public void description(String description) {
+        this.description = description;
+    }
+
+    public String target() {
+        return target;
+    }
+
+    public void target(String target) {
+        this.target = target;
+    }
+
+    public String adminGroup() {
+        return adminGroup;
+    }
+
+    public void adminGroup(String adminGroup) {
+        this.adminGroup = adminGroup;
+    }
+
+    public String opGroup() {
+        return opGroup;
+    }
+
+    public void opGroup(String opGroup) {
+        this.opGroup = opGroup;
+    }
+
+    public String name() {
+        return name;
+    }
+
+    public void name(String name) {
+        this.name = name;
+    }
+
+    public List<String> graphs() {
+        return graphs;
+    }
+
+    public void graphs(List<String> graphs) {
+        this.graphs = graphs;
+    }
+
+    public String graph() {
+        return graph;
+    }
+
+    public void graph(String graph) {
+        this.graph = graph;
+    }
+
+    @Override
+    public String type() {
+        return HugeType.PROJECT.string();
+    }
+
+    @Override
+    public Date createTime() {
+        return this.create;
+    }
+
+    @Override
+    public Date updateTime() {
+        return this.update;
+    }
+
+    @Override
+    public String creator() {
+        return this.creator;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
the code style expect "{}"

Reply:
ok,  equals and hashcode method are generated by intellij idea, i will check and fix it;